app/controllers/ThresholdInTwelveMonthsController.scala

```diff
@@ -48,14 +48,14 @@ class ThresholdInTwelveMonthsController @Inject()(mcc: MessagesControllerCompone
 
   def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
-      (request.userAnswers.fixedEstablishment, request.userAnswers.nino, request.userAnswers.businessEntity) match {  //TODO Move the journey continue logic into a separate controller: SAR-8201
-        case (Some(true), Some(true), _) =>
+      (request.userAnswers.fixedEstablishment, request.userAnswers.registeringBusiness, request.userAnswers.businessEntity) match {  //TODO Move the journey continue logic into a separate controller: SAR-8201
+        case (Some(true), _, _) =>
```

Comment:
The 2nd field in this case and the one on line 58 should be Some(_) as we want to ensure that the user has gone through the registering business page.